src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -252,3 +254,152 @@ def test_WINNF_FT_S_SCS_15(self,config_filename):
 
     # Check registration response
     self.assertEqual(response['response']['responseCode'],104)
+
+  def generate_SCS_17_default_config(self, filename):
+    """Generates the WinnForum configuration for SCS_17. """
+    # Create the configuration for short lived invalid client cert/key,wait timer
+
+    config = {
+      'clientCert': self.getCertFilename("short_lived_client.cert"),
```

Comment:
The cert should be created during the test case execution. Otherwise ITS will have to do some manual steps to run this test case (and any manual step for them increasing our total testing time disproportionately, since now it required them to monitor the tests much more closely and coordinate them more carefully).

Reply:
Added   creation of certificate before test case execution